src/components/Dashboard/Alerts/FrozenInformation.vue

```diff
@@ -0,0 +1,110 @@
+<template>
+  <v-expansion-panels v-model="panel" id="frozen-information-container">
+    <v-expansion-panel class="mb-6">
+      <v-expansion-panel-header hide-actions class="d-flex justify-space-between px-6 py-5">
+        <h3>
+          <v-icon left color="orange darken-2">mdi-alert</v-icon>
+          <span>This business is frozen</span>
+        </h3>
+        <v-btn text color="primary" class="details-btn my-n1" @click.stop="togglePanel()">
+          <span color="primary">{{ panel === 0 ? "Hide Details" : "View Details" }}</span>
```

Comment:
Not sure why we're using integer instead of boolean?

Reply:
Because it's the panel number -- see line 2.